lib/modules/manager/npm/__snapshots__/utils.spec.ts.snap

```diff
@@ -0,0 +1,43 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`modules/manager/npm/utils composeLockFile composes lockfile string out of an object 1`] = `
+"{
+  \\"lockfileVersion\\": 2,
+  \\"name\\": \\"lockfile-parsing\\",
+  \\"packages\\": {
+    \\"\\": {
+      \\"license\\": \\"ISC\\",
+      \\"name\\": \\"lockfile-parsing\\",
+      \\"version\\": \\"1.0.0\\"
+    }
+  },
+  \\"requires\\": true,
+  \\"version\\": \\"1.0.0\\"
+}"
+`;
+
+exports[`modules/manager/npm/utils parseLockFile can deal with invalid lockfiles 1`] = `
+Object {
+  "detectedIndent": "  ",
+  "lockFileParsed": undefined,
+}
+`;
+
+exports[`modules/manager/npm/utils parseLockFile parses lockfile string into an object 1`] = `
+Object {
+  "detectedIndent": "  ",
+  "lockFileParsed": Object {
+    "lockfileVersion": 2,
+    "name": "lockfile-parsing",
+    "packages": Object {
+      "": Object {
+        "license": "ISC",
+        "name": "lockfile-parsing",
+        "version": "1.0.0",
+      },
+    },
+    "requires": true,
+    "version": "1.0.0",
+  },
+}
+`;
```

Comment:
Please use `toStrictEqual` matcher instead of snapshot for those small objects.

Reply:
would you inline the strings too? I mean the fixture file I created and the first string snapshot.

Reply:
Pushed a version that moves only the objects out of the snapshots, hard-coding the strings gets weird fast I assume. Feel free to add more comments/unresolve again and sorry for the noise :)